### ISSUE TITLE
Simple post search has WCAG A violation due to empty list

### ISSUE THREAD

**Issue by michaelchadwick** [ilios/iliosproject.org#261]
The search results use a `<ul>` but it's empty if no search has been done, which is a a11y violation.
